src/microbe_stage/OrganelleLayout.cs

```diff
@@ -234,4 +235,62 @@ IEnumerator IEnumerable.GetEnumerator()
     {
         return Organelles.GetEnumerator();
     }
+
+    /// <summary>
+    ///   Recursively loops though all hexes and checks if there any without connection to the rest.
```

Comment:
The comment here and in CheckmarkNeighbours should be updated as this is no longer recursive.